hazelcast/src/test/java/com/hazelcast/jmx/MBeanTest.java

```diff
@@ -124,6 +124,15 @@ public void testExecutor() throws Exception {
         holder.assertMBeanExistEventually("IExecutorService", executor.getName());
     }
 
+    @Test
+    public void testReplicatedMap() throws Exception {
```

Comment:
Are there no properties on the mbean that can be tested? Currently only a check for existence is done.